test/parallel/test-repl-require-context.js

```diff
@@ -13,7 +13,7 @@ child.stdout.on('data', (data) => {
 });
 
 child.on('exit', common.mustCall(() => {
-  const results = output.replace(/^> /mg, '').split('\n');
+  const results = output.replace(/^> /mg, '').split('\n').filter((v, i) => i > 1);
```

Comment:
Nit: `Array#slice` would be better here, IMO.

Reply:
Fixed.